src/PointwiseFunctions/SpecialRelativity/LorentzBoostMatrix.hpp

```diff
@@ -0,0 +1,54 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+
+/// \cond
+namespace gsl {
+template <typename T>
+class not_null;
+}  // namespace gsl
+
+/// \endcond
+
+/// \ingroup SpecialRelativityGroup
+/// Holds functions related to special relativity.
+namespace sr {
+// @{
+/*!
+ * \ingroup SpecialRelativityGroup
+ * \brief Computes the matrix for a Lorentz boost from a single
+ * velocity vector (i.e., not a velocity field).
+ *
+ * \details Given a spatial velocity vector \f$v^i\f$ (with \f$c=1\f$),
+ * compute the matrix \f$\Lambda^{a}_{\bar{a}}\f$ for a Lorentz boost with
+ * that velocity [e.g. Eq. (2.38) of \cite ThorneBlandford2017]:
+ *
+ * \f{align}{
+ * \Lambda^t{}_{\bar{t}} &= \gamma, \\
+ * \Lambda^t_{\bar{i}} = \Lambda^i{}_{\bar{t}} &= \gamma v^i, \\
+ * \Lambda^i_{\bar{j}} = \Lambda^j{}_{\bar{i}} &= [(\gamma - 1)/v^2] v^i v^j
```

Comment:
Also, I'm confused why the free indices don't agree between the left and right side here.

Reply:
Good question, @nilsdeppe! See the equation and surrounding discussion that I cite. The equations are written here exactly as in Thorne and Blandford, and I think their way of writing these equations is reasonable.

These matrices $\Lambda^a{}_{b}$ transform between two different reference frames defined by two different sets of Lorentz (Cartesian) coordinates, $x^a$ and $x^{\bar{a}}$. In special relativity (flat spacetime) in Cartesian coordinates, spatial indices may be freely raised and lowered, because the spatial metric is just $\delta_{ij}$. So the indices of $n^j$ and $\delta^{ij}$ may be freely raised and lowered as needed here.

Also, these equations are specifying components of a matrix, not components of a tensor. As defined here, the indices on $\Lambda^a{}_{b}$ and its inverse, $\Lambda^{\bar{a}}{}_{b}$, are chosen so that barred and unbarred indices balance when transforming the coordinates of tensors with the corresponding index valences, e.g. $A^{\bar{a}}{}_{\bar{b}\bar{c}} = \Lambda^{\bar{a}}{}_{a} \Lambda^{b}{}_{\bar{b}} \Lambda^{c}{}_{\bar{c}} A^{a}{}_{b}$. The equations as written are just a shorthand for spelling out each matrix component in terms of three components of the spatial velocity, $v^x, v^y, v^z$, so that the expected transformations hold.

Reply:
Thanks for explaining that! I think my question wasn't clear. The LHS has Lambda^i_bar{j} and the RHS has v^i v^j. So for i = 1, which j do I choose? Should there be a delta_j^bar{j} on the RHS?